nemo_skills/pipeline/convert.py

```diff
@@ -219,14 +233,28 @@ def convert(
     if convert_to in ["hf", "nemo"] and model_type == "deepseek_v3":
         raise ValueError("Conversion to HF/Nemo is not yet supported for DeepSeek v3 models")
 
+    # Prepare cluster config and mount paths
     cluster_config = get_cluster_config(cluster, config_dir)
-    check_if_mounted(cluster_config, input_model)
-    check_if_mounted(cluster_config, output_model)
+    cluster_config = resolve_mount_paths(cluster_config, mount_paths)
+
+    if check_mounted_paths:
+        if not is_mounted_filepath(cluster_config, input_model): add_mount_path(input_model, "/input_model", cluster_config)
```

Comment:
best not to use single line ifs, harder to read and also our current black setup in precommit will reformat this back to 2 lines anyway. And in general, best to setup precommit to ensure your code is automatically formatted